db/migrate_cards/20171110164955_customizable_skin.rb

```diff
@@ -0,0 +1,85 @@
+# -*- encoding : utf-8 -*-
+
+class CustomizableSkin < Card::Migration::Core
+  def up
+    data_dir = File.expand_path("../data/custom_theme", __FILE__)
+    skin = CustomSkin.new("customizable bootstrap", data_dir)
+    skin.create_or_update
+  end
+
+  class CustomSkin
+    THEME_FIELDS = %w[colors components spacing cards fonts more]
```

Comment:
wow!

Reply:
This is old code. It's here for the bootstrap upgrade. There is a bug in the migrations. Migrations in core mods don't run. I didn't want to deal with that now so I copied the migration to wikirate.